radicle-cli/src/commands/inspect.rs

```diff
@@ -149,10 +149,9 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     } else if options.history {
         let repo = storage.repository(id)?;
         let head = Doc::<Untrusted>::head(signer.public_key(), &repo)?;
-        let history = repo.revwalk(head)?.collect::<Vec<_>>();
-        let revision = history.len();
+        let history = repo.revwalk(head)?.collect::<Vec<_>>().into_iter();
```

Comment:
@cloudhead what is the reason to collect into a vector?  Are the iterators between revwalk() and Vec<_> different?

Reply:
Actually that's no longer needed; as we collected to be able to reverse the iteration direction, but we no longer reverse, so could be removed.